.github/workflows/translations.yml

```
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: Update Translations
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: ["main", "2.4", "2.5"]
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4.1.7
+        with:
+          ref: ${{ matrix.branch }}
+
+      - name: Install Dependencies
+        run: sudo apt-get install -y --no-install-recommends -- qt6-l10n-tools
+
+      - name: Install transifex client
+        run: curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+
+      - name: Pull translations
+        run: ./tx pull -a -f --minimum-perc 1
+        env:
+          TX_TOKEN: ${{ secrets.TRANSIFEX_TOKEN }}
+
+      - name: Compile translations
+        run: for XX in res/translations/mixxx_*.ts; do /usr/lib/qt6/bin/lrelease -nounfinished $XX -qm ${XX/%.ts/.qm}; done
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          branch: pull-translations/${{ matrix.branch }}
+          commit-message: |-
+            Pull Transifex translations for ${{ matrix.branch }}
```

```suggestion
            chore(translations): Pull ${{ matrix.branch }} branch from Transifex
```

---

super nit-y, but [commitlint](https://github.com/conventional-changelog/commitlint) enforce the subject case to start with lower case by default. I know we don't use this linter and that `subject-case` is highly controversial,s o happy to keep it as is, but I think it's common to use lower cased subject

```suggestion
            chore(translations): pull ${{ matrix.branch }} branch from Transifex
```

---

Are you

> super nit-y, but [commitlint](https://github.com/conventional-changelog/commitlint) enforce the subject case to start with lower case by default.

Are you sure about that? To me it looks like Start Case, Sentence case, lower case and UPPER CASE are allowed by default: https://commitlint.js.org/reference/rules.html#subject-case
The actual Conventional Commit spec does not enforce any specific case for the subject: https://www.conventionalcommits.org/en/v1.0.0/#are-the-types-in-the-commit-title-uppercase-or-lowercase

Personally I'm in favor of `Sentence case` for readability (and also because all lowercase looks kind of lazy to me xD).